src/content/blog/astro-5.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: "Astro 5.0"
+description: "Astro 5.0 brings exciting new features like the Astro Content Layer for seamless content organization from any source, and Server Islands for effortlessly combining static and dynamic personalized content."
+publishDate: "2024-12-03T00:00:00+00:00"
+authors:
+  - erika
+  - matt
+  - matthew
+coverImage: "/src/content/blog/_images/astro-500/blog-post-5.webp"
+socialImage: "/src/content/blog/_images/astro-500/og-astro-5.webp"
+lang: "en"
+homepageLink:
+  title: "Astro 5.0"
+  subtitle: "is here!"
```

Comment:
```suggestion
  subtitle: "available now!"
```
I don't hate "is here", but here's a suggestion if you want!

Reply:
Yeah, hesitated between the two, they both work and they're both part of our pre-defined list